RR_Secondary Analysis_Bar plot + kw test.R

```diff
@@ -0,0 +1,102 @@
+chooseCRANmirror()
+install.packages("AER")
+install.packages("lmtest")
+
+# load the necessary libraries-------
+library(miscTools)
+library(libcoin)
+library(survival)
+library(dplyr)
+library(coin)
+library(readxl)
+library(dplyr)
+library(censReg)
+library(maxLik)
+library(ggplot2)
+library(lmtest)
+library(sandwich)
+library(zoo)
+library(AER)
```

Comment:
I think only packages that are used in this script are `{readxl}`, `{dplyr}` and `{ggplot2}`